### ISSUE TITLE
the problem of empty vars expressions in "Advanced Routing Matching Conditions"

### ISSUE THREAD

**Issue by Firstsawyou** [apache/apisix-dashboard#1798]
# Feature request

## Please describe your feature

When adding a route, Dashboard will add an empty expression of `"vars":{}` by default:
![image](https://user-images.githubusercontent.com/52862365/115331938-6d594f00-a1c9-11eb-9640-f0e3bbcef968.png)

However, only in the APISIX 2.5 version, APISIX will perform schema verification on the `"vars":{}` expression in the route. If there is `"vars":{}` in the route, the following error message will be reported:

```
2021/04/14 14:09:03 [error] 42#42: 1 [lua] config_etcd.lua:230: load_full_data(): failed to check item data of [/apisix/routes] err:failed to validate the 'vars' expression: rule too short ,val: {"uris":["/ecom/"],"vars":{},"hosts":["dev.url","develop.url"],"id":"350417588708180914","update_time":1618401644,"status":1,"create_time":1618394758,"name":"ecom_dev_v2","upstream":{"hash_on":"vars","scheme":"http","type":"roundrobin","timeout":{"read":6000,"connect":6000,"send":6000},"pass_host":"pass","nodes":[{"host":"ecom-edg.development.svc.cluster.local","weight":1,"port":8085,"priority":0}]},"priority":0,"methods":["GET","POST","PUT","DELETE"]}, context: init_worker_by_lua*
```

I see that the community Dashboard 2.6 is ready to be released, which will be used in conjunction with the APISIX 2.5 version. But I am not sure whether Dashboard 2.6 will add the empty expression of `"vars":{}` by default when adding routes. If there is, then maybe Dashboard 2.6 and APISIX 2.5 are used together, and the above-mentioned incompatibility problem may occur. 

Back-end related issues: https://github.com/apache/apisix/issues/4048

## Describe the solution you'd like

If Dashboard version 2.6 adds the empty expression of `"vars":{}` by default when adding routes, then we should remove it.

**Comment by belongwqz** [comment]
does Dashboard version 2.6 fix this error?

**Comment by Firstsawyou** [comment]
> does Dashboard version 2.6 fix this error?

This is a known issue of APISIX 2.5, and it is incompatible with data created in earlier versions of Dashboard 2.5. This problem has been fixed in later versions of APISIX 2.5.

https://github.com/apache/apisix/issues/4048
https://github.com/apache/apisix/pull/4056

**Comment by juzhiyuan** [comment]
@belongwqz no, wait for #1799
